pkg/dashboard/apiserver/common/common.go

```diff
@@ -262,11 +271,12 @@ func (s *Service) getKinds(c *gin.Context) {
 // MapSlice defines a common map
 type MapSlice map[string][]string
 
-// @Summary Get the labels of the pods in the specified namespace from Kubernetes cluster.
-// @Description Get the labels of the pods in the specified namespace from Kubernetes cluster.
+// @Summary Get the labels of the pods/physicalMachines in the specified namespace from Kubernetes cluster.
+// @Description Get the labels of the pods/physicalMachines in the specified namespace from Kubernetes cluster.
 // @Tags common
 // @Produce json
-// @Param podNamespaceList query string true "The pod's namespace list, split by ,"
+// @Param podNamespaceList query string false "The pod's namespace list, split by ,"
+// @Param physicalMachineNamespaceList query string false "The physicalMachine's namespace list, split by ,"
 // @Success 200 {object} MapSlice
 // @Router /common/labels [get]
 // @Failure 500 {object} utils.APIError
```

Comment:
I prefer to create another webapi endpoint for fetching labels of `physicalMachines`.

The current implementation is not good, if `podNamespaceList` is set and `len(podNamespaceList) > 0`, it would only query labels for physicalMachine. The parameters design for this API is bad. 🥲

Reply:
And how about using another PR for the changes on Chaos Dashboard? 🙈

Reply:
> I prefer to create another webapi endpoint for fetching labels of physicalMachines.

Actually I have thought about this question, it let me confused. The only one reason I choose this implementation is that the original API called `/common/labels` but not `/common/podLabels`, if I add a new API `/common/physicalMachineLabels`, it seems a bit weird. And I agree that this implementation is not so good, so I'll add a new API as you suggested :)

> The current implementation is not good, if podNamespaceList is set and len(podNamespaceList) > 0, it would only query labels for physicalMachine. 

BTW, in this implementation, I add some judgements for two params, if `podNamespaceList` is set, the API will query for pod, I'm not sure what you mean here (but it's not important now).

> And how about using another PR for the changes on Chaos Dashboard?

Ooops, I'd really want to put it in another pr, but the speed that waiting for this pr to be merged seems not very fast, and I need to show the API for frontend developer first (so he could start this work early), so I decided to put the API in this pr together 😢 .

Reply:
done